internal/playlist/list.go

```diff
@@ -0,0 +1,30 @@
+package playlist
+
+import (
+	"fmt"
+	"spotify/internal"
+
+	"github.com/spf13/cobra"
+)
+
+func NewListCommand() *cobra.Command {
+	return &cobra.Command{
+		Use:     "list",
+		Aliases: []string{"l"},
```

Comment:
This isn't a command people will be running repeatedly, so no need for an alias in my opinion.
```suggestion
```